<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskImportExport</class>
 <widget class="QWidget" name="TaskImportExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>758</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>1</y>
     <width>850</width>
     <height>750</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>相机→电脑</string>
    </attribute>
    <widget class="QPushButton" name="btnExport">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>440</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>421</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>详细信息</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>描述:</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>创建人:</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="txtAuthor">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_TaskInfo">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>281</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="txt_Name">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="txt_Date">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>日期:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>名称:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>420</y>
       <width>281</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>电脑地址</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ExportIP">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>36</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.0.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>261</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>任务文件列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_TaskFileName"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>电脑→相机</string>
    </attribute>
    <widget class="QPushButton" name="btnImport">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>300</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>导入</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>300</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>电脑地址</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_ImportIP">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.0.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>34</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>400</width>
       <height>120</height>
      </rect>
     </property>
     <property name="title">
      <string>任务名称</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_ImportName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>290</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>35</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>名称:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>注：任务名称不支持中文</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>相机→U盘</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_TaskFileName_U">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>58</y>
       <width>259</width>
       <height>432</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>31</y>
       <width>259</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>任务文件列表</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>421</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>详细信息</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>描述:</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>创建人:</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="txtAuthor_U">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_TaskInfo_U">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>281</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="txt_Name_U">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_U">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="txt_Date_U">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>日期:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>名称:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnExport_U">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>470</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>U盘→相机</string>
    </attribute>
    <widget class="QPushButton" name="btn_U_Import">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>360</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>导入</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_TaskFileName_U_Import">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>57</y>
       <width>259</width>
       <height>651</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>259</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>U盘任务文件列表</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_TaskFileName_Camera">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>57</y>
       <width>259</width>
       <height>651</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>259</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>相机任务文件列表</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
